frontend/src/Notifications.jsx

```diff
@@ -26,15 +41,15 @@ const StyledMenu = withStyles({
   },
 })((props) => (
   <Menu
-    elevation={0}
+    //elevation={0}
```

Comment:
Deze comment even weghalen als hij niet meer gebruikt wordt. Als het mogelijk in de toekomst wel weer gebruikt moet worden is het handig om hier een comment boven te zetten met een TODO + wat er gedaan moet worden